ruby/src/api.cc

```diff
@@ -136,11 +136,7 @@ namespace leatherman { namespace ruby {
         } else {
             LOG_INFO("ruby was already loaded.");
         }
-        try {
-            return unique_ptr<api>(new api(move(library)));
-        } catch (lth_lib::missing_import_exception& ex) {
-            throw ex;
-        }
+        return unique_ptr<api>(new api(move(library)));
```

Comment:
The error is coming from line 139 though... no auto involved. Though that did fix line 125.